src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -1,24 +1,43 @@
-import * as React from 'react'
-import { MDXRenderer } from 'gatsby-plugin-mdx'
-import { graphql, Link } from 'gatsby'
-import AuthorsYAMLData from "../../content/authors.yaml"
-import CategoriesYAMLDATA from "../../content/categories.yaml"
+import * as React from 'react';
+
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { graphql, Link } from 'gatsby';
+
+import AuthorsYAMLData from "../../content/authors.yaml";
+import CategoriesYAMLDATA from "../../content/categories.yaml";
+import { Category } from "../components/category";
+
+import * as styles from './blog-post.module.scss';
+
 const _ = require("lodash")
 
 const BlogPost = ({ data }) => {
-  const author = AuthorsYAMLData.find(author => { return author.author === data.mdx.frontmatter.author })
-  const category = CategoriesYAMLDATA.find(category => { return category.category === data.mdx.frontmatter.category })
+  const author = AuthorsYAMLData.find(author => { return author.author === data.mdx.frontmatter.author });
+  const category = CategoriesYAMLDATA.find(category => { return category.category === data.mdx.frontmatter.category });
+
   return (
-    <>
-        <div><Link to={`/categories/${ _.kebabCase(category.category) }`}>{ category.title }</Link></div>
-        <div><Link to={`/authors/${ _.kebabCase(author.author) }`}>{ author.display_name }</Link></div>
-        <div>{ author.image }</div>
-        <div>{ author.profile_url }</div>
-        <p>{data.mdx.frontmatter.date}</p>
-        <MDXRenderer>
-            {data.mdx.body}
-        </MDXRenderer>
-    </>
+    <div className={styles.container}>
+      <div className={styles.headerContainer}>
+        <Category data={category.display_name}/>
+        <div className={styles.titleContainer}>
+          { data.mdx.frontmatter.title }
+        </div>
+        <div className={styles.authorContainer}>
+          <div className={styles.authorInformation}>
+            <div className={styles.authorImage}>
+              <img src={`../../static/images/${author.image}`} />
+            </div>
```

Comment:
I don't agree to use a div for only one component as a child, maybe you can move the authorImage and apply it in the className of the img